### ISSUE TITLE
Rich text is incorrectly removing aria- and data- attributes from source html on save

### ISSUE THREAD

**Issue by natechadwick** [percussion/percussioncms#1042]
**Describe the bug**

When editing and saving Rich text widget content, any aria or data attributes are incorrectly getting stripped from the content on save .

**To Reproduce**

1. Go to a Page with a Rich Text widget.
2. Go to Source Code view and add the following code:
``` html
<i class="fa fa-file" aria-hidden="true"></i>
```
3. Click ok on Source View and Save the content. 
4. Re-open the Rich text widget and go to source code view.  Notice that the aria-hidden="true" is missing. 

**Expected behavior**
The rich text editor should not be stripping out aria or data attributes.

**Versions Affected**
* 8.1.2
